overrides/main.html

```diff
@@ -1,23 +1,13 @@
 {% extends "base.html" %}
 
-<!-- Custom front matter -->
-{% block extrahead %}
-
-<!-- Extra style sheets (can't be set in mkdocs.yml due to content hash) -->
-<link rel="stylesheet" href="{{ 'assets/stylesheets/custom.css' | url }}" />
-{% endblock %}
-
 <!-- Announcement bar -->
 {% block announce %}
-<span style="font-size: larger">We've moved! You're looking at the new HGVS Nomenclature pages.</span>
+<span style="font-size: larger">We've moved! You're looking at the new HGVS Nomenclature pages. <a href="/versions/21.0/">What's Changed?</a></span>
```

Comment:
This link is also broken, for the same reason. Since it also won't work on the stable site, perhaps we should remove it? Or we should fix the issue properly.

Reply:
So annoying! Having relocatable sites but need the ability to link to documents relative to the root seems nearly incompatible. I'm stuck on this one too. I'll remove this and the versions link.

Reply:
Indeed :thinking:  I usually use the `base` element in my websites to solve this, but that would be too much of a hassle now. We can try to solve it later.

Reply:
I think this is the same as previous. I vote to leave as-is. We can't meaningfully test it until we merge this branch.

Reply:
This link now also works!